test/boost/database_test.cc

```diff
@@ -409,14 +409,12 @@ SEASTAR_THREAD_TEST_CASE(test_distributed_loader_with_pending_delete) {
 
     const sstring toc_text = "TOC.txt\nData.db\n";
 
-    std::optional<sstables::generation_type> prev_gen;
     std::vector<sstables::generation_type> gen;
-    size_t num_gens = 9;
-    gen.reserve(num_gens);
-    for (auto i = 0; i < num_gens; i++) {
+    size_t NUM_GENS = 9;
```

Comment:
nit: by convention we use all caps for constant definitions.
I'd keep `num_gens` here in lower case (maybe it should even be randomized as a follow up).
Currently, as a constant it be be defines as `constexpr`.

Reply:
okay. i don't feel strong either way. just felt that it's a constant. will

1. keep it lowercase
2. add `constexpr`